src/SDKs/ContainerRegistry/dataplane/Microsoft.Azure.ContainerRegistry/Microsoft.Azure.ContainerRegistry.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.ContainerRegistry</PackageId>
+    <Description>Provides management capabilities for Azure Container Registry.</Description>
+    <AssemblyTitle>Microsoft Azure Container Registry</AssemblyTitle>
+    <Version>0.9.0-preview</Version>
+    <AssemblyName>Microsoft.Azure.ContainerRegistry</AssemblyName>
+    <PackageTags>Microsoft Azure Container Registry;Container Registry;</PackageTags>
+    <PackageReleaseNotes>Added capabilities to manage repository, tag and manifest.</PackageReleaseNotes>
```

Comment:
This should be release notes about the SDK being published rather than the commit.
For example
```
This is a public preview SDK for Microsoft.Azure.ContainerRegistry services
```

Reply:
Shouldn't the release notes list the change included in this release? The developers already know this is preview sdk through the package versioning.

Reply:
Ohh, I guess I understood it incorrectly, this SDK allows one to manage a repo.